src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  let result = sourceString.split(';');
+
+  result = result.filter((item) => item !== '');
+  result = result.map((item) => item.split(':').map((elem) => elem.trim()));
+
+  result = result.reduce((acc, item) => {
+    acc[item[0]] = item[1];
+
+    return acc;
+  }, {});
+
+  return result;
```

Comment:
The function currently does not handle cases with multiple semicolons without any styles between them. While it's not required to handle this case, it would be a good practice to ensure the function is robust against various input formats.